app/controllers/admin/users_controller.rb

```diff
@@ -124,7 +124,7 @@ def suspend
       on_success do
         render_json_dump(
           suspension: {
-            suspend_reason: result.reason,
+            suspend_reason: result.contract.reason,
```

Comment:
It would probably be interesting to be able to do something like
```ruby
on_success do |contract:, user:|
```
to access directly the various objects needed here. One more item added to my todo list :sweat_smile: